.travis.yml

```diff
@@ -14,3 +14,11 @@ install:
   - echo y | yo rn-toolbox:eslint
   - echo y | yo rn-toolbox:jest
   - echo GeneratorTest | yo rn-toolbox:base --force
+  - cd ..
+before_script:
+  - npm prune
```

Comment:
i got `yarn not found`. I kinda wanna get this money now and setup a proper testing CI later :)